components/eam/src/dynamics/se/stepon.F90

```diff
@@ -232,7 +234,7 @@ subroutine stepon_run2(phys_state, phys_tend, dyn_in, dyn_out )
    use cam_history,     only: outfld, hist_fld_active
    use prim_driver_base,only: applyCAMforcing_tracers
    use prim_advance_mod,only: applyCAMforcing_dynamics
-   use element_ops,     only: get_temperature
+   use element_ops,     only: get_temperature 
```

Comment:
Please revert extra whitespace change here to clean up diffs.

Reply:
cleaned